session_items.py

```diff
@@ -67,3 +67,16 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def remove_item(item):
+    """
+    Removes an existing item in the session. If no existing item matches the ID of the specified item, nothing is removed.
```

Comment:
Technically this checks if the item's attributes match (so just having the same id does not suffice)